proposals/nnnn-actors.md

```diff
@@ -359,15 +307,113 @@ extension BankAccount {
     await modifiesAsynchronously(&balance) // error: actor-isolated property 'balance' cannot be passed 'inout' to an asynchronous function
   }
 }  
+
+class C { var state : Double }
+struct Pair { var a, b : Double }
+actor A {
+  let someC : C
+  var somePair : Pair
+
+  func inoutModifications() async {
+    modifiesSynchronously(&someC.state)       // okay
+    await modifiesAsynchronously(&somePair.a) // not okay
+    modifiesSynchronously(&someC.state)       // okay
+    await modifiesAsynchronously(&somePair.a) // not okay
```

Comment:
lines 320-1 repeat lines 318-9; should the later pair swap their arguments to show that synchronous modification of both `class` and `struct` members are okay, but asynchronous modification is disallowed on both?

Thank you for further examples of how classes and structs interoperate with actors, by the way :)

Reply:
Yes, that was the goal. I'll push a fix, thank you!